src/tests/ftest/launch.py

```diff
@@ -1003,6 +1091,94 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+def check_big_files(avocado_logs_dir, task, test_name, args):
+    """Check the contents of the task object, tag big files, create junit xml.
+
+    Args:
+        avocado_logs_dir (str): path to the avocado log files.
+        task (Task): a Task object containing the command result
+        test_name (str): current running testname
+        args (argparse.Namespace): command line arguments for this program
+
+    Returns:
+        bool: True if no errors occurred checking and creating junit file.
+            False, otherwise.
+
+    """
+    status = True
+    hosts = NodeSet()
+    cdata = []
+    for output, nodelist in task.iter_buffers():
+        node_set = NodeSet.fromlist(nodelist)
+        hosts.update(node_set)
+        big_files = re.findall(r"Y:(.*)", str(output))
+        if big_files:
+            cdata.append(
+                "The following log files on {} exceeded the {} "
+                "threshold:".format(node_set, args.logs_threshold))
+            cdata.extend(["  {}".format(big_file) for big_file in big_files])
+    if cdata:
+        destination = os.path.join(avocado_logs_dir, "latest")
+        status = create_results_xml(
+            hosts, test_name, "\n".join(cdata), destination)
+    else:
+        print("No log files found exceeding {}".format(args.logs_threshold))
+
+    return status
+
+
+def create_results_xml(hosts, testname, output, destination):
+    """Create Junit xml file.
+
+    Args:
+        hosts (list): list of host names
+        testname (str): name of test
+        output (dict): result of the command.
+        destination (str): directory where junit xml will be created
+
+    Returns:
+        bool: status of writing to junit file
+
+    """
+    status = True
+    testsuite_attrs = {
+        "name": "{}".format(testname),
+        "errors": "1",
+        "failures": "0",
+        "skipped": "0",
+        "test": "1",
+        "time": "0.0",
+    }
+    testcase_attrs = {
+        "name": "ALL",
```

Comment:
Maybe we should use this field to help identify this error in the Jenkins Test Results output, i.e.
```suggestion
        "name": "framework_results",
```

Reply:
Will do